Index: app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignoutView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignoutView.java b/app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignOutView.java
rename from app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignoutView.java
rename to app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignOutView.java
--- a/app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignoutView.java	
+++ b/app/src/main/java/ca/utoronto/cscb07project/ui/loginsignout/LoginSignOutView.java	
@@ -1,7 +1,8 @@
 package ca.utoronto.cscb07project.ui.loginsignout;
 
-public interface LoginSignoutView {
+public interface LoginSignOutView {
     void successfulSignIn();
     void failedSignIn();
+
     void SignOut();
 }
